### PR TITLE
Upload task

### PR DESCRIPTION
**Pull request by ymilenin**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import dao.HumanDaoImpl;
+import lib.Injector;
+
+public class Main {
+
+    static {
+        try {
+            Injector.injectDependency();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
```
Comment (SevostyanovGerman) [inline src/Main.java]: cs

File: src/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    return;
+                }
+                String[] data = command.split(" ");
+                int value = Integer.parseInt(data[0]);
+                double risk = Double.parseDouble(data[1]);
+                Bet bet = new Bet(value, risk);
+                betDao.add(bet);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (SevostyanovGerman) [inline src/controller/ConsoleHandler.java]: cs

File: src/dao/BetDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package dao;
+
+import db.Storage;
+import lib.Dao;
+import model.Bet;
+
+import java.util.List;
+
+@Dao
+public class BetDaoImpl implements BetDao {
```
Comment (SevostyanovGerman) [inline src/dao/BetDaoImpl.java]: cs

File: src/dao/HumanDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package dao;
+
+import db.Storage;
+import lib.Dao;
+import model.Human;
+
+import java.util.List;
+
+@Dao
+public class HumanDaoImpl implements HumanDao {
```
Comment (SevostyanovGerman) [inline src/dao/HumanDaoImpl.java]: cs

File: src/factory/BetDaoFactory.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+package factory;
+
+import dao.BetDao;
+import dao.BetDaoImpl;
+
+public class BetDaoFactory {
+
+    private static BetDao instance;
```
Comment (SevostyanovGerman) [inline src/factory/BetDaoFactory.java]: @Inject?

File: src/dao/BetDaoImpl.java (hunk 3 above)
Comment (ymilenin) [reply src/dao/BetDaoImpl.java]: ?

File: src/dao/HumanDaoImpl.java (hunk 4 above)
Comment (ymilenin) [reply src/dao/HumanDaoImpl.java]: ?

File: src/controller/ConsoleHandler.java (hunk 2 above)
Comment (ymilenin) [reply src/controller/ConsoleHandler.java]: IDEA tells me that cs is good

File: src/Main.java (hunk 1 above)
Comment (ymilenin) [reply src/Main.java]: IDEA tells me that cs is good

File: src/dao/BetDaoImpl.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    @Override
+    public void add(Bet bet) {
+        Storage.bets.add(bet);
+    }
+
+    @Override
+    public List<Bet> getAll() {
+        return Storage.bets;
+    }
+}
```
Comment (Sofasmile) [inline src/dao/BetDaoImpl.java]: the same

File: src/factory/HumanDaoFactory.java [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    private HumanDaoFactory() {
+    }
+
+    public static HumanDao getHumanDao() {
+        if (instance == null) {
+            return instance = new HumanDaoImpl();
+        }
+        return instance;
+    }
+}
```
Comment (Sofasmile) [inline src/factory/HumanDaoFactory.java]: forgot here)

**Pull request merged**